.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
This limit of 80 chars per line takes me back right to the 80s ... seriously? It's 2016. Let me use my larger-than-standard-VGA-resolution screen properly.